File: spring-cloud-integrated-demo/order-service/src/main/java/com/github/jccode/springcloud/integrateddemo/order/Application.java
package com.github.jccode.springcloud.integrateddemo.order;

import com.github.jccode.springcloud.integrateddemo.common.config.MicroServiceConfig;
import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
import org.springframework.cloud.netflix.eureka.EnableEurekaClient;
import org.springframework.cloud.netflix.feign.EnableFeignClients;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Import;

@SpringBootApplication
@Import(MicroServiceConfig.class)
public class Application {

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }

}

File: spring-cloud-integrated-demo/order-service/src/main/scala/com/github/jccode/springcloud/integrateddemo/order/Application.scala
package com.github.jccode.springcloud.integrateddemo.order

import com.github.jccode.springcloud.integrateddemo.common.config.MicroServiceConfig
import feign.Logger
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.{Bean, Configuration, Import}

@SpringBootApplication
@Import(Array(classOf[MicroServiceConfig]))
class Application {

}

object Application {
  def main(args: Array[String]): Unit = {
    SpringApplication.run(classOf[Application], args: _*)
  }
}


// Feign config log level
//   - Don't forget change the log level in application.yml as well.
@Configuration
class FeignCustomConfig {

  @Bean
  def feignLoggerLevel: Logger.Level = Logger.Level.FULL

}